### PR TITLE
Fix MELPA Stable & Contributing links in README

### PR DESCRIPTION
**Pull request by maio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ you prefer to only receive updates for tagged releases, use
 
 ## Contributing
 
-See the [CONTRIBUTING.md][] document.
```
Comment (milkypostman) [inline README.md]: im not sure I like this change as I would like to point out that the file name is part of the repo and is called CONTRIBUTING.md
 I'll commit this without this change. Thanks for pointing out the problem with the other link.

Comment (maio) [inline README.md]: `[CONTRIBUTING.md][]` is invalid Markdown. It renders as:

image

So if you want to keep it, then it should be `[CONTRIBUTING.md](CONTRIBUTING.md)`

Comment (milkypostman) [inline README.md]: whoops, works in some flavors.

I submitted 1ca9af77af79dfcee3811e6ffba60e53859113c2 to clean things up.

Comment (milkypostman) [inline README.md]: and b94357e35825c25816b71cdc3ce0fe161cef0ac0

**Comment by milkypostman** [comment]
i think i fixed everything let me know if I missed something. thanks for the pull request!

**Pull request closed without merging**